drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/condition/BRLConditionInspector.java

```diff
@@ -21,37 +21,48 @@
 public class BRLConditionInspector
         extends ConditionInspector {
 
-    public BRLConditionInspector( final CompositeColumn<? extends BaseColumn> column ) {
-        super( new BRLConditionInspectorKey( column ) );
+    private String valueAsString;
+
+    public BRLConditionInspector(final CompositeColumn<? extends BaseColumn> column,
+                                 final String valueAsString) {
+        super(new BRLConditionInspectorKey(column));
+        this.valueAsString = valueAsString;
     }
 
     @Override
-    public boolean conflicts( final Object other ) {
+    public boolean conflicts(final Object other) {
+        if (other instanceof BRLConditionInspector) {
+            final String otherValue = ((BRLConditionInspector) other).valueAsString;
+            if (otherValue != null) {
+                return !otherValue.equals(valueAsString);
```

Comment:
Could I ask you for explanation when should the method `conflicts` return `true` and when `false`.